data/avro-schema/src/main/resources/avro/net/corda/data/ledger/utxo/token/selection/data/StrategyAvro.avsc

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "enum",
+  "name": "StrategyAvro",
```

Comment:
We don't normally add "Avro" to the name here. We name the Avro class like we normally would, then import it with a different name for the source files that need both Avro and Corda types.

Here is one example:
```kotlin
import net.corda.ledger.utxo.verification.TransactionVerificationStatus as TransactionVerificationStatusAvro
```

Not saying that's the best approach, but we should aim for consistency. Otherwise we will have multiple approaches in our code and things will get confusing.